aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -232,10 +232,29 @@
       "type": "string",
       "description": "The Active Directory directory ID to create the DB instance in. Currently, only MySQL, Microsoft SQL Server, Oracle, and PostgreSQL DB instances can be created in an Active Directory Domain."
     },
+    "DomainAuthSecretArn": {
+      "type": "string",
+      "description": "The ARN for the Secrets Manager secret that contains the credentials for the user performing the domain join."
```

Comment:
Here as well
`The ARN for the Secrets Manager secret with the credentials for the user joining the domain.`